### PR TITLE
Edited old csharp syntax, removed "out" and replaced with return tuples

### PR DESCRIPTION
**Pull request by RedTaylor**

### CODE REVIEW

**Review by externl — APPROVED** [review]
Looks good to me. I would update the title to be something more descriptive describing what the commit actually does:

eg. Convert Slice out parameters to return types in some C# tests

File: csharp/test/Ice/enums/TestI.cs [hunk 1]
```diff
@@ -28,68 +28,68 @@ public class TestIntf : ITestIntf
         public (IEnumerable<SimpleEnum>, IEnumerable<SimpleEnum>) OpSimpleSeq(SimpleEnum[] s1, Current current) =>
             (s1, s1);
 
-        public (FLByteEnum ReturnValue, FLByteEnum B2) OpFLByte(FLByteEnum b1, Current current) => (b1, b1);
+        public (FLByteEnum R1, FLByteEnum R2) OpFLByte(FLByteEnum b1, Current current) => (b1, b1);
```
Comment (bernardnormier) [inline csharp/test/Ice/enums/TestI.cs:31]: I would simply leave the tuple members nameless, like in other tests.

File: csharp/test/Ice/enums/Test.ice [hunk 2]
```diff
@@ -198,40 +198,40 @@ sequence<MyFlags> MyFlagsSeq;
...
+    (FLSimpleEnum r1, FLSimpleEnum r2) opFLSimple(FLSimpleEnum s1);
+
+    (FLByteEnumSeq r1, FLByteEnumSeq r2) opFLByteSeq(FLByteEnumSeq b1);
+    (FLShortEnumSeq r1, FLShortEnumSeq r2) opFLShortSeq(FLShortEnumSeq s1);
+    (FLUShortEnumSeq r1, FLUShortEnumSeq r2) opFLUShortSeq(FLUShortEnumSeq s1);
+    (FLIntEnumSeq r1, FLIntEnumSeq r2) opFLIntSeq(FLIntEnumSeq i1);
+    (FLUIntEnumSeq r1, FLUIntEnumSeq r2) opFLUIntSeq(FLUIntEnumSeq i1);
+    (FLSimpleEnumSeq r1, FLSimpleEnumSeq r2) opFLSimpleSeq(FLSimpleEnumSeq s1);
+
+    (tag(1) ByteEnum? r1, tag(3) ByteEnum? r2) opTaggedByte(tag(2) ByteEnum? b1);
```
Comment (bernardnormier) [inline csharp/test/Ice/enums/Test.ice:225]: Can we use tag(1) and tag(2) for the return type members, and tag(1) for the parameter?

**Review by bernardnormier — APPROVED** [review]
Looks good.

**Review by InsertCreativityHere — APPROVED** [review]

**Review by externl — APPROVED** [review]

**Pull request merged**